suite/tests/security-common/selfmod.c

```diff
@@ -402,16 +484,61 @@ ADDRTAKEN_LABEL(immediate_addr_plus_four:)
         pop      REG_XDX
         pop      REG_XBP
         ret
+#else
+ADDRTAKEN_LABEL(sandbox_cross_page_no_ilt:)
+        stp x29, x30, [sp, #-16]!
+        /* Do enough writes to cause the sandboxing code to split the block. */
+        strb w0, [x1], #1
+        strb w0, [x1], #1
+        strb w0, [x1], #1
+        strb w0, [x1], #1
+
+        /* modify the "movz x9, #0" instruction */
+        adr x10, immediate_addr_plus_four - 4
+        ldr w11, [x10]
+        /* set the imm16 part of the movz instruction to w0 */
+        and w11, w11, #0xffe0001f /* set imm16 to 0 */
+        orr w11, w11, w0, LSL #5
```

Comment:
Add a comment explaining: is this writing to the immediate?  So it was set to 0 to set the 0 bits and this sets the 1 bits of the immedate?